firefox-ios/Client/Coordinators/Browser/BrowserCoordinator.swift

```diff
@@ -137,8 +137,20 @@ class BrowserCoordinator: BaseCoordinator,
         screenshotService.screenshotableView = nil
     }
 
-    func showHomepage() {
-        let homepageController = self.homepageViewController ?? HomepageViewController(windowUUID: windowUUID)
+    func showHomepage(overlayManager: OverlayModeManager, isZeroSearch: Bool) {
+        let homepageCoordinator = childCoordinators[HomepageCoordinator.self] ?? HomepageCoordinator(
+            windowUUID: windowUUID,
+            profile: profile,
+            wallpaperManger: WallpaperManager(),
```

Comment:
If we are creating a `WallpaperManager()` here, would it be worth it to move this as a default initializer in the `HomepageCoordinator` init?